parsers/verilog.c

```diff
@@ -1171,27 +1172,66 @@ static int skipParameterAssignment (int c)
 static int processFunction (tokenInfo *const token, int c)
 {
 	verilogKind kind = token->kind;	// K_FUNCTION or K_TASK
+#define TOKEN_NAME_SIZE 16
+	vString *tokenNameSaved[TOKEN_NAME_SIZE];
+	int doubleColonPos[TOKEN_NAME_SIZE];
```

Comment:
This is a really good idea. I really need to dynamically grow the array, otherwise I need to make a good trade-off with the array length. I don't know if there is already an implementation. I just searched in the project, so this form of writing is a copy from `./main/parse.c`